doc/faq.md

```diff
@@ -18,3 +18,22 @@ Please note that this file may contain sensitive information such as your IMSI a
 ### Should I get a locked or unlocked orbic device? What is the difference?
 
 If you want to use a non-Verizon SIM card you will probably need an unlocked device. But it's not clear how locked the locked devices are nor how to unlock them, we welcome any experimentation and information regarding the use of unlocked devices.
+
+
+### How do I re-enable USB tethering after installing Rayhunter?
+
+Make sure USB tethering is also enabled in the Orbic's UI, and then run the following commands:
+
+```sh
+installer util serial "echo 9 > /usrdata/mode.cfg"
```

Comment:
I think this should be `installer util serial "AT+SYSCMD='echo 9 > /usrdata/mode.cfg'" and same for reboot, no?